monkey/monkey_island/cc/ui/src/components/contexts/plugins/PluginsContext.tsx

```diff
@@ -1,36 +1,109 @@
-import React, {createContext, useState} from 'react';
+import React, {createContext, useEffect, useState} from 'react';
 import AuthComponent from '../../AuthComponent';
+import semver from 'semver';
+import _ from 'lodash';
 
 export const PluginsContext = createContext({});
 
 const authComponent = new AuthComponent({});
 
 export const PluginState = () => {
-
+  const [allPlugins, setAllPlugins] = useState([]);
+  const [allInstalledPlugins, setAllInstalledPlugins] = useState([]);
   const [availablePlugins, setAvailablePlugins] = useState([]);
   const [installedPlugins, setInstalledPlugins] = useState([]);
```

Comment:
It's very hard to tell what's the difference between all of these. Ideally, we should define types. 
I don't see why we need to store the unparsed data.

Reply:
PR will come